webapp/controller/App.controller.js

```diff
@@ -55,14 +57,18 @@ sap.ui.define([
 		 * Trigger search for specific items.
```

Comment:
could you update the documentation and state that, completed items cannot be removed while a filter is active (itemsRemovable flag).